client/templates/course/lesson/quiz/questions/autoform.js

```diff
@@ -0,0 +1,26 @@
+
+var addQuestionFormHooks = {
+    before: {
+        method: function(doc, params){
+            var editedQuestion = Session.get("currentQuestionToBuild");
+            editedQuestion.title = doc.title;
+            editedQuestion.description = doc.description;
+            return editedQuestion;
+        }
+    },
+    onSuccess: function(operation, result, template) {
+
+        //clear the session cache of the currently edited question
+        Session.set("currentQuestionToBuild", undefined);
+        Session.set("currentQuestionTypeChanged", undefined);
+        delete Session.keys["currentQuestionToBuild"];
+        delete Session.keys["currentQuestionTypeChanged"];
+        return sAlert.success("question added");
```

Comment:
The `return` statement may be unnecessary here. Simply using `sAlert...` may suffice.